src/browser/HostInstaller.cpp

```diff
@@ -139,8 +142,8 @@ void HostInstaller::installBrowser(SupportedBrowsers browser,
  */
 void HostInstaller::updateBinaryPaths(const bool& proxy, const QString& location)
 {
-    // Where 6 is the number of entries in the SupportedBrowsers enum declared in HostInstaller.h
-    for (int i = 0; i < 6; ++i) {
+    // Where 7 is the number of entries in the SupportedBrowsers enum declared in HostInstaller.h
```

Comment:
comment is not accurate with the current change

Reply:
Recommend removing this comment entirely.